src/js/redux/reducers/search/searchFiltersReducer.js

```diff
@@ -99,6 +100,11 @@ const searchFiltersReducer = (state = initialState, action) => {
                         state.selectedRecipientLocations, action.location)
             });
         }
+        case 'UPDATE_TEXT_SEARCH': {
```

Comment:
Can you move this to the top of the `switch` statement, adding an appropriate comment above, so this statement can match the order of the Sidebar filters?